block/algebraic/petsc/solver.py

```diff
@@ -0,0 +1,26 @@
+from block.block_base import block_base
+from petsc4py import PETSc
+
+class LU(block_base):
```

Comment:
@jobh Is the intent here that the method only works with matrices that are not `block_mat`?

Reply:
Hm, good question. 

I think I will just delete this PR, it seems to be misguided. There's already LU and MUMPS_LU in preconditioners.